tests/functional/aws-node-sdk/test/object/putObjectLegalHold.js

```diff
@@ -1,164 +1,164 @@
-const assert = require('assert');
-const { errors } = require('arsenal');
-const withV4 = require('../support/withV4');
-const BucketUtility = require('../../lib/utility/bucket-util');
-const removeObjectLock = require('../../lib/utility/objectLock-util');
+// const assert = require('assert');
```

Comment:
from a change perspective a single block comment might be easier here

Reply:
oh yea that's right 😅